src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -288,57 +288,61 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets() {
 
         // First input branch of MatMul0 should be executed before second input branch of MatMul0,
         // so firstly we insert Transpose1 on the beginning of ordered_ops and then Transpose1
```

Comment:
so firstly we insert **Transpose1** on the beginning of ordered_ops and then **Transpose1**
I assume one of them must be Transpose0 🙃 

BTW It's clear what Matmul0 and Matmul1 are, but it's much less obvious where Transpose0 and Transpose1 are located. One may think that Transpose1 is connected to Matmul1 for example. So my suggestion here is either to reflect transposes on the scheme on L216-227, or at least to state somewhere smth like: Transpose0 and Transpose1 are connected to the 0th and 1st inputs of Matmul0.